modules/updating-sno.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-between-minor.adoc
+
+= Updating single node {product-title}
+
+You can update, or upgrade, a single-node {product-title} cluster using either the console or CLI methods. 
+
+However, because single node {product-title} does not support high availability, the following prerequisites are not required: 
+
+* having a recent etcd backup
+* pausing the `MachineHealthCheck` resources
+
+Upgrading single node {product-title} requires downtime and can include an automatic reboot. The amount of downtime depends on the upgrade payload: 
+
+* If the update payload contains an operating system update, which requires a reboot, the downtime is significant and impacts cluster management and user workloads. 
+
+* If the update has machine configuration changes, but does not require a reboot, the downtime is less and the impact on the cluster management and user workloads is lessened. In this case the nodes are not drained, because there is no other node in the cluster to reschedule the workloads.
```

Comment:
It will be more clear to mention like this "node drain is not performed on Single Node (or skipped) for changes requiring reboot or a MachineConfig change that would have performed drain on HA cluster, because there is no other node in the cluster to reschedule the workloads"